tox.ini

```diff
@@ -2,13 +2,30 @@
 envlist = py38-sphinx{11,14},docs,quality
 
 [doc8]
-max-line-length = 120
-
+; D001 = Line too long
+ignore=D001
 [pep8]
```

Comment:
@jinder1s: You lost the empty line here.

Reply:
fixed